deploy/charts/venafi-kubernetes-agent/README.md.gotmpl

```diff
@@ -6,134 +6,130 @@
+First create an RSA key pair:
 
 ```shell
-export VENAFI_NAMESPACE="venafi" VENAFI_SERVICE_ACCOUNT="example-cluster"
+export VENAFI_SERVICE_ACCOUNT="example-cluster"
 openssl genrsa -out ${VENAFI_SERVICE_ACCOUNT}.pem
 openssl rsa -in ${VENAFI_SERVICE_ACCOUNT}.pem -pubout --out ${VENAFI_SERVICE_ACCOUNT}.pub
 ```
 
-Now that you have both the private and public key we now need to use the Venafi Control Plane to create a service account.
+Next create a service account in the Venafi Control Plane:
 
-- Navigate to the [service accounts page](https://ui.venafi.cloud/service-accounts/) and select "New"
-- Add a unique name matching the variable we used, eg: "example-cluster"
-- Assign a team that owns this credential
+- Click **Settings > Service Accounts**.
+- Click **New**.
+- Type a name for your new service account.
+  Must match the ${VENAFI_SERVICE_ACCOUNT} variable that you used above.
+- Select an **Owning Team**, which is the team who owns the machine you want to create the service account for.
 - The scope should be "Kubernetes Discovery" only.
 - Set the validity period of your pubic key up to a maximum of 365 days.
-- Now paste in the **public key** from the pair you generated.
+- Paste in the **public key** from the pair you generated.
+- Click **Save** to finish and return to the Service Account list view.
+- Find the row matching the name you entered and copy the "Client ID" value,
+  by clicking "Copy Client ID" in the row actions menu.
+  You will need this when you install the Helm chart.
 
-Once created, you will be returned to the service accounts list.
-Find your newest entry matching the name you entered, and copy the "Client ID" column.
+### 2) Deploy the chart
 
-### 3) Deploying the chart:
-
-Now we have the service account, let us prepare a namespace with the relevant private key needed at runtime.
+Create a Namespace and then create a Secret containing the private key of the service account:
 
 ```shell
-export VENAFI_CLIENT_ID="<PASTE YOURS HERE>"
+export VENAFI_NAMESPACE="venafi"
 kubectl create namespace ${VENAFI_NAMESPACE}
-kubectl create secret generic agent-credentials -n ${VENAFI_NAMESPACE} \
+kubectl create secret generic agent-credentials \
+  --namespace ${VENAFI_NAMESPACE} \
   --from-file=privatekey.pem=${VENAFI_SERVICE_ACCOUNT}.pem
 ```
 
-Install the chart by setting the `config.clientId` field:
+Install the chart:
 
 ```shell
-helm upgrade --install venafi-kubernetes-agent deploy/charts/venafi-kubernetes-agent \
+export VENAFI_CLIENT_ID="<your-client-id>"
+helm upgrade venafi-kubernetes-agent oci://registry.venafi.cloud/charts/venafi-kubernetes-agent \
+  --install \
   --namespace ${VENAFI_NAMESPACE} \
   --set config.clientId="${VENAFI_CLIENT_ID}"
 ```
 
-Optionally if you need to change the backend to the EU Venafi Control Plane you can use:
+> To change the backend to the EU Venafi Control Plane, use the following Helm value:
+> `--set config.server="${VENAFI_SERVER_URL}"`
```

Comment:
We haven't exported this variable previously, so maybe we include it, or just add this into the instructions if this might change often? Not sure what's best, but also happy to leave how it is and assume people know who to export the env var and what it should be exported to.

Reply:
I've hard coded the EU URL here instead.